x-pack/plugins/cases/public/components/create/tags.test.tsx

```diff
@@ -47,45 +48,71 @@ describe('Tags', () => {
   beforeEach(() => {
     jest.clearAllMocks();
     useGetTagsMock.mockReturnValue({ data: ['test'] });
+    appMockRender = createAppMockRenderer();
   });
 
   it('it renders', async () => {
-    const wrapper = mount(
+    appMockRender.render(
       <MockHookWrapperComponent>
         <Tags isLoading={false} />
       </MockHookWrapperComponent>
     );
 
     await waitFor(() => {
-      expect(wrapper.find(`[data-test-subj="caseTags"]`).exists()).toBeTruthy();
+      expect(screen.getByTestId('caseTags')).toBeInTheDocument();
     });
   });
 
-  it('it disables the input when loading', async () => {
-    const wrapper = mount(
+  it.skip('it disables the input when loading', async () => {
```

Comment:
after removing mount, this test doesn't pass anymore. In UI also I am not able to reproduce the behaviour(disabled combobox) while loading 🤷‍♀️

Reply:
Ok, let's remove it.